tensorflow-core/tensorflow-core-api/build.sh

```diff
@@ -24,6 +24,7 @@ fi
 
 if [[ "${EXTENSION:-}" == *gpu* ]]; then
     export BUILD_FLAGS="$BUILD_FLAGS --config=cuda"
+    export TF_CUDA_COMPUTE_CAPABILITIES=3.5,sm_52,sm_60,sm_61,7.0,7.5
```

Comment:
Why the mix of `3.5` and `sm_52` formats? Why not `3.5` style or `sm_35` style for everything?

Reply:
Trying to reduce the binary size according to https://github.com/tensorflow/tensorflow/blob/master/configure.py#L1003

>Each capability can be specified as "x.y" or "compute_xy" to include both virtual and binary GPU code, or as "sm_xy" to only include the binary code.\n Please note that each additional compute capability significantly increases your build time and binary size

Reply:
Ok, so that means it'll fallback to 3.5 level compute if it hits a non 5.2 Maxwell or a non 6.0 Pascal. Not sure why it's got both Turing and Volta levels though. From the docs it suggests it uses `sm_35,sm_37,sm_52,sm_60,sm_61,7.0` - https://www.tensorflow.org/install/gpu as it says it's only got PTX for the latest supported architecture. Where did you get the list you used from?

Reply:
From one of the committer comments when I research about the issue and I can't seem to find it now.
Updated to the same as official website doc. Thanks!

Reply:
The latest version of the official binaries for Python don't include PTX code. It's probably not a good idea to follow what they are doing there. They'll probably revert that change once they get flooded with enough duplicate issues about that.

Reply:
We should poke someone to update the docs page to note they've removed the PTX from the binaries (though I agree it'll be back for compatibility reasons).